src/register-org/components/payment-by-account/payment-by-account.component.html

```diff
@@ -4,41 +4,37 @@
       <a class="govuk-back-link" href="javascript:void(0)" (click)="onBack()">Back</a>
       <xuilib-hmcts-error-summary [errorMessages]="validationErrors" [header]="'There is a problem'"></xuilib-hmcts-error-summary>
       <span class="govuk-caption-xl">Register an organisation</span>
+      <h1 class="govuk-heading-xl">
```

Comment:
@johnbenjamin-hmcts - same comment as earlier on use of `<fieldset>` with radio buttons

Reply:
@Daniel-Lam - fieldset is used for grouping controls together, whereas in this case it is only Yes No.

https://design-system.service.gov.uk/components/fieldset/

Reply:
@Daniel-Lam - I have updated the code now.